docs/fastdds/dds_layer/topic/contentFilteredTopic/writerFiltering.rst

```diff
@@ -6,3 +6,24 @@
 Where is filtering applied: writer vs reader side
 =================================================
 
+Content filters may be evaluated on either side, as the DataWriter obtains the filter expression from the
+DataReader during discovery.
+
+Filtering on the writer side can save network bandwidth at the cost of increasing CPU usage on the writer.
+
+.. _dds_layer_topic_contentFilteredTopic_writer_restrictions:
+
+Conditions for writer side filtering
+------------------------------------
+
+A DataWriter will perform filter evaluation for a DataReader when all of the following conditions are met.
+Filtering will otherwise be performed by the DataReader.
+
+- The DataWriter has infinite liveliness. See |LivelinessQosPolicy|.
+- Communication with the DataReader is neither intra-process nor data-sharing.
+- The DataReader is not using multicast.
```

Comment:
Reference to the section where how to disable multicast traffic is explained.

Reply:
User DataReaders do not use multicast by default. So I don't think that section is relevant

Reply:
Done in 60fa349c16c9833b5fb8e6cdff516cef593a5748

Reply:
Done in 60fa349c16c9833b5fb8e6cdff516cef593a5748